doc/examples/transform/plot_transform_types.py

```diff
@@ -0,0 +1,146 @@
+"""
+===================================
+Types of homographies
+===================================
+
+`Homographies <https://en.wikipedia.org/wiki/Homography>`_
+are transformations of a Euclidean space that preserve the alignment of points.
+Specific cases of homographies correspond to the conservation of more properties,
+such as parallelism (affine transformation), shape (similar transformation) or
+distances (Euclidean transformation).
+
+Homographies on a 2D Euclidean space (i.e., for 2D grayscale or multichannel
+images) are defined by a 3x3 matrix. All types of homographies can be defined
+by passing either the transformation matrix, or the parameters of the simpler
+transformations (rotation, scaling, ...) which compose the full transformation.
+
+The different types of homographies available in scikit-image are
+shown here, by increasing order of complexity (i.e. by reducing the number of constraints). While we focus here on the
+mathematical properties of transformations, tutorial
+:ref:`sphx_glr_auto_examples_transform_plot_geometric.py` explains how to use
+such transformations for various tasks such as image warping or parameter
+estimation.
+"""
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage import transform as tf
```

Comment:
I personally don't like this abbreviation since `tf` is usually used for `tansorflow`